sdks/java/io/solace/src/main/java/org/apache/beam/sdk/io/solace/data/Solace.java

```diff
@@ -17,26 +17,26 @@
  */
 package org.apache.beam.sdk.io.solace.data;
 
+import com.google.auto.value.AutoValue;
+import com.solacesystems.jcsmp.BytesXMLMessage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import org.apache.beam.sdk.schemas.AutoValueSchema;
+import org.apache.beam.sdk.schemas.annotations.DefaultSchema;
+import org.apache.beam.sdk.schemas.annotations.SchemaFieldNumber;
+import org.apache.beam.vendor.grpc.v1p60p1.com.google.protobuf.ByteString;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
- * A record to be written to a Solace topic.
- *
- * <p>You need to transform to {@link Solace.Record} to be able to write to Solace. For that, you
- * can use the {@link Solace.Record.Builder} provided with this class.
- *
- * <p>For instance, to create a record, use the following code:
- *
- * <pre>{@code
```

Comment:
Is deleting this  code snippet intended, (like it has moved to other places)?

Reply:
It was intended. The old snippet of the documentation was more suitable for the Write connector, not the Read.